aks/postgres/outputs.tf

```diff
@@ -26,15 +26,16 @@ output "name" {
 }
 
 output "url" {
-  value     = "postgres://${urlencode(local.database_username)}:${urlencode(local.database_password)}@${local.host}:${local.port}/${local.database_name}"
+  value     = "postgres://${urlencode(local.database_username)}:${urlencode(local.database_password)}@${local.host}:${local.port}/${local.database_name}?sslmode=${var.use_azure ? "require" : "disable"}"
```

Comment:
Can we use prefer instead of disable, to always try SSL first

Reply:
fixed